sdk/identity/azure-identity/azure/identity/_credentials/browser.py

```diff
@@ -66,20 +70,45 @@ class InteractiveBrowserCredential(InteractiveCredential):
             :caption: Create an InteractiveBrowserCredential.
     """
 
-    def __init__(self, **kwargs: Any) -> None:
-        redirect_uri = kwargs.pop("redirect_uri", None)
+    def __init__(
+        self,
+        *,
+        authority: Optional[str] = None,
+        tenant_id: Optional[str] = None,
+        client_id: Optional[str] = None,
+        login_hint: Optional[str] = None,
+        redirect_uri: Optional[str] = None,
+        authentication_record: Optional["AuthenticationRecord"] = None,
+        disable_automatic_authentication: Optional[bool] = None,
+        cache_persistence_options: Optional["TokenCachePersistenceOptions"] = None,
+        timeout: Optional[int] = None,
```

Comment:
Why not timeout: int = 300, ?

Reply:
Was initially trying to keep the typing the same as what Apiview has, but makes sense for it to be an `int` with a default.